src/Frontend/Page/Overview.hs

```diff
@@ -130,9 +130,13 @@ instance ToHtml PageIdeasInDiscussion where
                            , href_ . U.listIdeas $ IdeaLocationTopic space (topic ^. _Id)
                            ] $ do
                             img_ [ src_ . U.TopStatic $ "images" </> case topic ^. topicPhase of
-                                      PhaseRefinement _ -> "theme_aus.png"
+                                      PhaseWildIdea     -> "theme_aus.png"  -- FIXME
+                                      PhaseWildFrozen   -> "theme_aus.png"  -- FIXME
```

Comment:
Yep, that's what I have in old commits in this PR, but then I needed to add the placeholders due to tests (and in other places), as I reported during the standup.